modules/aws_asg/main.tf

```diff
@@ -265,11 +265,15 @@ resource "aws_autoscaling_group" "workers" {
 }
 
 resource "aws_autoscaling_policy" "workers_remove_capacity" {
-  name                   = "${var.env}-${var.index}-workers-${var.site}-remove-capacity"
-  scaling_adjustment     = "${var.worker_asg_scale_in_qty}"
-  adjustment_type        = "ChangeInCapacity"
-  cooldown               = "${var.worker_asg_scale_in_cooldown}"
-  autoscaling_group_name = "${aws_autoscaling_group.workers.name}"
+  name                      = "${var.env}-${var.index}-workers-${var.site}-remove-capacity"
+  adjustment_type           = "ChangeInCapacity"
+  policy_type               = "StepScaling"
```

Comment:
I'm confused about our current scaling policy being "simple" given the terraform docs about `policy_type` 😕 https://www.terraform.io/docs/providers/aws/r/appautoscaling_policy.html#policy_type which indicate we're using `StepScaling` whether or not we specify it.  Sure enough, source says: https://github.com/hashicorp/terraform/blob/5f24491ea6659c50883d85a063f902a22b8c2a50/builtin/providers/aws/resource_aws_appautoscaling_policy.go#L40-L44